README.md

```diff
@@ -98,9 +98,9 @@ protected function init()
 
 We use `themedCSS` and `themedJavascript` to auto-locate the resource based on the current theme. That way, if the theme ever changes, we don't have to update the path.
 
-The only resources we haven't included are the html5 shim that is conditionally included for IE8, and the Google font that is loaded from an absolute path.
+The only resource we haven't included is the html5 shim that is conditionally included for IE8, and the Google font that is loaded from an absolute path.
```

Comment:
I reckon it fits better the way it was - thoughts?

Reply:
It's a single resource (one file), referenced in the singular (shim, not shims), that's why I altered it.

Reply:
Does the google font not count as a resource? It does in my head

Reply:
lol, brain, how do you even work? Totally just skipped over that xD
Reminds me of those posters that go:

```
This was silly and
and rather illogical
```